core/src/main/java/org/apache/calcite/sql/SqlSelect.java

```diff
@@ -262,6 +267,10 @@ public void setFetch(SqlNode fetch) {
     this.fetch = fetch;
   }
 
+  public final SqlNode getExceptExpression() {
```

Comment:
Let's replace the getter with a public final field.

Reply:
Will do, I wasn't sure which to have in this case as there looked to be a lot of existing getters of this form.

Reply:
I agree, it's confusing because the original developers of Calcite didn't even follow a consistent convention themselves on this. However, Julian's recommendation to me was to use fields rather than getters, so we'll just do that going forward.